datahub-graphql-core/src/test/java/com/linkedin/datahub/graphql/resolvers/ingest/secret/UpdateSecretResolverTest.java

```diff
@@ -0,0 +1,101 @@
+package com.linkedin.datahub.graphql.resolvers.ingest.secret;
+
+import static com.linkedin.datahub.graphql.resolvers.ingest.IngestTestUtils.getMockAllowContext;
+import static com.linkedin.datahub.graphql.resolvers.ingest.IngestTestUtils.getMockDenyContext;
+import static com.linkedin.metadata.Constants.SECRET_VALUE_ASPECT_NAME;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.when;
+
+import com.datahub.authentication.Authentication;
+import com.linkedin.common.AuditStamp;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.common.urn.UrnUtils;
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.exception.AuthorizationException;
+import com.linkedin.datahub.graphql.generated.UpdateSecretInput;
+import com.linkedin.entity.Aspect;
+import com.linkedin.entity.EntityResponse;
+import com.linkedin.entity.EnvelopedAspect;
+import com.linkedin.entity.EnvelopedAspectMap;
+import com.linkedin.entity.client.EntityClient;
+import com.linkedin.metadata.secret.SecretService;
+import com.linkedin.r2.RemoteInvocationException;
+import com.linkedin.secret.DataHubSecretValue;
+import graphql.schema.DataFetchingEnvironment;
+import java.net.URISyntaxException;
+import java.util.concurrent.CompletionException;
+import org.mockito.Mockito;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class UpdateSecretResolverTest {
+
+  private static final Urn TEST_URN = UrnUtils.getUrn("urn:li:secret:secret-id");
+
+  private static final UpdateSecretInput TEST_INPUT =
+      new UpdateSecretInput(TEST_URN.toString(), "MY_SECRET", "mysecretvalue", "dummy");
+
+  private DataFetchingEnvironment mockEnv;
+  private EntityClient mockClient;
+  private SecretService mockSecretService;
+  private UpdateSecretResolver resolver;
+
+  @BeforeMethod
+  public void before() throws RemoteInvocationException, URISyntaxException {
+    mockClient = Mockito.mock(EntityClient.class);
+    mockSecretService = Mockito.mock(SecretService.class);
+
+    resolver = new UpdateSecretResolver(mockClient, mockSecretService);
+  }
+
+  private DataHubSecretValue createSecretAspectWithTypeUrn(Urn testUrn) {
```

Comment:
Suggestion: Rename helper method to match what it does: `createSecretAspectWithUrn`